### ISSUE TITLE
No package matching 'origin-node-3.11*

### ISSUE THREAD

**Issue by sklakken** [gshipley/installcentos#99]
Hi,

I've been able to install OpenShift using this repo consistently until now. It appears that version 3.11 is not available to the general public (see below). I tried downloading it directly with no luck. 

Please advise...

Recreate:

1) Install base version of CentOS 7 with networking and git installed.
2) Simply run ./install-openshift.sh

```
******
* Your domain is <domain> 
* Your IP is <ip address>
* Your username is <username>
* Your password is <password>
* OpenShift version: 3.11 
******
```
Error Message:

```
fatal: [192.168.0.101]: FAILED! => {"attempts": 3, "changed": false, "msg": "No package matching 'origin-node-3.11*' found available, installed or updated", "rc": 126, "results": ["No package matching 'origin-node-3.11*' found available, installed or updated"]}
```
Available 'Origin' packages

# yum --showduplicates list origin | expand
```
origin.x86_64        3.6.0-1.0.c4dd4cf                  centos-openshift-origin 
origin.x86_64        3.6.1-1.0.008f2d5                  centos-openshift-origin 
origin.x86_64        3.7.0-1.0.7ed6862                  centos-openshift-origin 
origin.x86_64        3.7.1-1.el7.git.0.0a2d6a1          centos-openshift-origin 
origin.x86_64        3.7.1-2.el7                        centos-openshift-origin 
origin.x86_64        3.8.0-1.el7.git.0.dd1558c          centos-openshift-origin 
origin.x86_64        3.9.0-1.el7.git.0.ba7faec          centos-openshift-origin 
origin.x86_64        3.10.0-1.el7.git.0.0c4577e         centos-openshift-origin 
```

**Comment by marekjelen** [comment]
Fixed in PR #97 ... waiting for @gshipley review

**Comment by oveits** [comment]
@marekjelen: True. It is also fixed in the more minimalistic PR #102. Maybe the review of that PR is quicker since there is less meat to it. However, if PR #97 is merged, PR #102 will become obsolete.
